@stellar/design-system/src/types/@modules.d.ts

```diff
@@ -5,3 +5,13 @@ declare module "*.svg" {
   const src: string;
   export default src;
 }
+
+declare module "react-copy-to-clipboard" {
+  import React = require("react");
+
+  export default React.PureComponent<{
+    children: React.ReactNode;
+    text: string;
+    onCopy?: () => void;
+  }>;
+}
```

Comment:
Defining these types ourselves because `@types/react-copy-to-clipboard` were causing weird issues.

Reply:
I can't wait for [clipboard api](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard#browser_compatibility) to be widely adopted so we can replace this with that :D

Reply:
:100: